src/components/ScenarioComponents/CardsComponents/CardContainer.tsx

```diff
@@ -99,48 +136,56 @@ export default function CardContainer({
     return `${height}px`;
   }, [compartmentsExpanded, maxCompartmentsRows, minCompartmentsRows]);
 
-  const dataCards = scenarios.map((scenario) => {
-    return (
-      <DataCard
-        key={scenario.id}
-        id={scenario.id}
-        color={scenario.color}
-        title={scenario.name}
-        compartmentsExpanded={compartmentsExpanded}
-        compartmentValues={cardValues ? cardValues[scenario.id] : null}
-        referenceValues={referenceValues ?? null}
-        selectedCompartmentId={selectedCompartmentId}
-        filterValues={filterValues}
-        isSelected={selectedScenario === scenario.id}
-        isActive={scenario.active}
-        setSelected={setSelectedScenario}
-        setActive={setActiveScenario}
-        remove={removeScenario}
-        minCompartmentsRows={minCompartmentsRows}
-        maxCompartmentsRows={maxCompartmentsRows}
-        localization={localization}
-        groupFilters={groupFilters}
-        arrow={arrow}
-      />
-    );
-  });
+  const dataCards = scenarios.map((scenario, index) => (
+    <DataCard
+      key={scenario.id}
+      id={scenario.id}
+      color={scenario.color}
+      title={scenario.name}
+      compartmentsExpanded={compartmentsExpanded}
+      compartmentValues={cardValues ? cardValues[scenario.id] : null}
+      referenceValues={referenceValues ?? null}
+      selectedCompartmentId={selectedCompartmentId}
+      filterValues={filterValues}
+      isSelected={selectedScenario === scenario.id}
+      isActive={scenario.active}
+      setSelected={setSelectedScenario}
+      setActive={setActiveScenario}
+      draggable={index !== 0 && index !== 1}
+      remove={removeScenario}
+      minCompartmentsRows={minCompartmentsRows}
+      maxCompartmentsRows={maxCompartmentsRows}
+      localization={localization}
+      groupFilters={groupFilters}
+      arrow={arrow}
+    />
+  ));
 
   return (
-    <Box
-      id='card-container'
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        gap: 4,
-        minHeight: minHeight,
-        overflowX: 'auto',
-        minWidth: 400,
-        paddingLeft: 4,
-        paddingRight: 4,
-        paddingTop: 2,
-      }}
+    <DndContext
+      sensors={sensors}
+      collisionDetection={closestCenter}
+      onDragEnd={handleDragEnd}
+      modifiers={[restrictToHorizontalAxis]}
     >
-      {dataCards}
-    </Box>
+      <SortableContext items={scenarios.map((s) => s.id)} strategy={horizontalListSortingStrategy}>
+        <Box
+          id='card-container'
+          sx={{
+            display: 'flex',
+            flexDirection: 'row',
+            gap: 4,
+            minHeight: minHeight,
+            overflowX: 'auto',
+            minWidth: 400,
+            paddingLeft: 4,
+            paddingRight: 4,
+            paddingTop: 2,
+          }}
```

Comment:
Make the box fill the parent container, so the scrollbar only appears when no further space is available.
```suggestion
          sx={{
            display: 'flex',
            flexDirection: 'row',
            flexGrow: 1,
            gap: 4,
            minHeight: minHeight,
            overflowX: 'auto',
            minWidth: 400,
            paddingLeft: 4,
            paddingRight: 4,
            paddingTop: 2,
          }}
```